package.json

```diff
@@ -22,36 +9,19 @@
     "react",
     "node"
   ],
-  "dependencies": {
-    "eslint": "^7",
-    "eslint-config-prettier": "^8",
-    "eslint-plugin-import": "^2",
-    "eslint-plugin-jest": "^24",
-    "eslint-plugin-prettier": "^4",
-    "eslint-plugin-promise": "^5",
-    "eslint-plugin-react": "^7",
-    "eslint-plugin-react-hooks": "^4",
-    "eslint-plugin-react-native": "^3",
-    "eslint-plugin-react-native-a11y": "^3",
-    "eslint-plugin-redux-saga": "^1",
-    "eslint-restricted-globals": "^0.2.0",
-    "prettier": "^2"
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "postinstall": "test -n \"$NOYARNPOSTINSTALL\" || husky install",
+    "sort-all-package-json": "yarn sort-package-json \"package.json\" \"packages/*/package.json\""
   },
   "devDependencies": {
-    "generate-changelog": "^1"
+    "@commitlint/cli": "^17.1.2",
+    "@commitlint/config-conventional": "^17.1.0",
+    "husky": "^8.0.1",
+    "lerna": "^4.0.0",
```

Comment:
We started to use Turborepo in our projects. So I think it would be nice to keep consistency and use it here as well.

If not, Lerna has released v5.6.x version, so we can update it to the latest one I think. https://github.com/lerna/lerna/releases